third_party/install.sh

```diff
@@ -92,12 +92,7 @@ CMAKE_OPTIONS="${CMAKE_OPTIONS} -DCMAKE_INSTALL_LIBDIR=lib"
 if [[ ${cmake_options} =~ USE_CAFFE=ON || ${cmake_options} =~ USE_ONNX=ON || ${cmake_options} =~ USE_FLOW=ON ]]; then
     PROTOC_ROOT=${work_dir}/protoc
     protobuf_version="3.14.0"
-    protobuf_cmake_options=""
-    if [[ ${target} =~ linux-aarch64 ]]; then
-        protobuf_version="3.1.0"
```

Comment:
when using linux-aarch64 run on android phone, we find that protobuf only 3.1.0 can work, we encounter error when using 3.14.